### DISCUSSION TITLE
Some Test it on Raspberry 5 ? || When we see a new Version on Github ?

### DISCUSSION THREAD

**Discussion by Holli82** [KevinBytesTheDust/CVAmp#226] [General]
Hey the Raspberry 5 is Out since some Weeks has some Test it with Wine maybe to Run it on some.
Where interess to know maybe how much it can Handle. 

Other Topic
I see that were Working on the Project but see that more and more Version just on Kofi.
When we get maybe a Short Version Update for the " twitch token" Login that brings bit more Effort to it.
Maybe a Raid Feature to Change the "URL" for the Proxys.. 

On thiss Way Thanks for this nice little Tool like how it works. Mostly Performe good and it has a non confuse UI no Special but literaly understandble for all kinde of Noobs. 👍

**Comment by jlplenio** [comment]
Thank you very much for your words. As for the Raspberry 5:
If you run the tool natively in Python, I would estimate that it can handle between 15-20 160p instances. If you use something like Wine, it will either not work at all or become unusable due to performance issues. To run it natively, you would have to use the command line in the root folder of the toll and:

1. Install python 3.9
2. Install poetry via `pip install poetry`
3. Install the packages via `poetry install"
4. Start the tool with `poetry run python main_gui.py`

Regarding ko-fi and GitHub, I am currently quite conflicted. Other users have linked me sites, where rebranded and modified versions of this tool are flat-out being sold for profit. This is the reason I have become more protective about the code that enables all the cool features we have discussed and implemented together (auth, chat, follow, auto-chat). I will discuss this over the Christmas holidays with my friend Chardonnay.

**Reply by user-t01** [reply]
Hello, going through the instructions I was almost able to run the tool through Python natively. My issue right now is that the code is currently trying to use Chrome instead of the built in chromium on the raspberry OS. Does someone have a solution for this?

log: https://pastebin.com/B5GqUNtG

**Comment by Holli82** [comment]
Hope you find good Solutions for the Protection and that they cant Steal your well Work. 
I hate People that Adorn themselves with other feathers and the main Shit is that they Sell it...

Thats not nice to hear but hope you find a solution who you can maybe find it easyer and has maybe some options to let them Delete this modified Versions. 

And Thanks for the Explain with the Raspberry 5 nice to hear and know that you have try it. A value between 15-20 is a good Value for the Pi, has run a little Laptop who i have Struckle to handle it with less. :D 

Have a good Chrismas Holiday and thanks for your quick reply
